styles/styles.css

```diff
@@ -0,0 +1,390 @@
+	font-family: "Open Sans", sans-serif;
+	font-weight: 700;
+	font-size: 1rem;
+	padding: 1rem 2rem;
+}
+
+.header-nav a:hover {
+	background-color: #253979;
+	color: white;
+}
+
+
+/* FOOTER STYLING */
+
+footer {
+	background-color: #253979;
+}
+
+.footer-elements {
+	padding: 4rem;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+.footer-logo {
+	width: 45%;
+}
+
+.footer-nav {
+	padding: 0;
+}
+
+.footer-nav li {
+	font-family: "Open Sans", sans-serif;
+	color: white;
+	display: inline-block;
+	margin: 1rem;
+}
+
+.footer-nav a {
+	text-decoration: none;
+	color: white;
+}
+
+.footer-elements p {
+	font-family: "Open sans", sans-serif;
+	font-size: 1rem;
+	width: 60%;
+	color: white;
+	margin-bottom: 0;
+	text-align: center;
+}
+
+small {
+	color: white;
+}
+
+
+/* INDEX CSS */
+
+main {
+	margin: 1rem 10rem;
+}
+
+.about-us {
+	margin-bottom: 6rem;
+	margin-top: 3rem;
+	display: grid;
+	/* grid-template-columns: 1fr 1fr; */
+	grid-template-columns: minmax(300px, .5fr) 1fr;
+	gap: 2.5rem;
+}
+
+.about-us-blurb {
+	align-self: center;
+	justify-self: center;
+}
+
+.about-us-blurb p {
+	font-size: 1.4rem;
+}
+
+.index-main {
+	display: grid;
+	grid-template-columns: 70% 1fr;
+	grid-gap: 4rem;
+}
+
+
+/* DOG CARD CSS */
+
+.dog-card-board {
+	display: grid;
+	grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+	gap: 3rem;
+}
+
+.od-card-board {
+	margin-bottom: 4rem;
+}
+
+.dog-card {
+	border: 2px solid #e8e8e8;
+	border-radius: 3px;
+	box-shadow: 0px 0px 10px -1px rgba(0, 0, 0, 0.20);
+}
+
+.dog-card-details {
+	padding: 1rem;
+}
+
+.dog-card>h3 {
+	margin-left: 1rem;
+}
+
+.dog-card-details>p:nth-of-type(1) {
+	display: inline;
+	font-weight: 700;
+}
+
+.dog-card-details>p:nth-of-type(2) {
+	display: inline;
+}
+
+.dog-card-details>button {
+	background-color: green;
+	font-size: 1rem;
+	font-family: "Open sans", sans-serif;
+	font-weight: 700;
+	color: white;
+	width: 100%;
+	padding-top: 10px;
+	padding-bottom: 10px;
+	border: none;
+}
+
+.see-more-dogs-button {
+	background-color: #E4E4E4;
+	font-size: 1rem;
+	font-family: "Open sans", sans-serif;
+	font-weight: 700;
+	color: #253979;
+	padding: 15px 30px;
+	border: none;
+	text-align: center;
+	display: block;
+	margin: 2rem auto 4rem auto;
+}
+
+.blog-container {
+	display: grid;
+	grid-template-columns: fit-content(100%) 1fr;
+	gap: 2rem;
+	margin: 3rem 0;
+}
+
+.blog-img {
+	width: 100%;
+}
+
+.blog-copy {
+	align-self: center;
+}
+
+.blog-copy p {
+	font-size: 1rem;
+	line-height: 1.75rem;
+}
+
+@media only screen and (max-width: 1350px) {
+	main {
+		margin: .5rem 1rem;
+	}
+	.index-main {
+		grid-gap: 2rem;
+	}
+	.about-us {
+		margin-top: 1.5rem;
+		margin-bottom: 1rem;
+		grid-template-columns: repeat(auto-fit, minmax(100%, 1fr));
+		grid-gap: 0;
+	}
+	.about-us-blurb p {
+		font-size: 1.2rem;
+	}
+	.blog-container {
+		grid-template-columns: repeat(auto-fit, minmax(400px, 1fr));
+		gap: 2rem;
+	}
+}
+
+@media only screen and (max-width: 850px) {
```

Comment:
technically I should be able to see the blog container at ipad with per the assignment. You could adjust this value to make that happen.